temporal.yaml

```diff
@@ -23,17 +23,19 @@ pipeline:
       repository: https://github.com/temporalio/cli
       tag: v${{package.version}}
       expected-commit: 7267df668277a9b43c50d454af50ff5e2b9f8e90
+  
+  - uses: patch
```

Comment:
Could we open a PR against the upstream repo with these changes? When you do it, please add a comment to this yaml def with a ref. Perhaps they can release a patch version soon. So we don't need to deal with patches here.

Reply:
Yes I will make a PR updating the go module with these changes.
even if it does not get merged there will be discussion about it.